scripts/start.sh

```
@@ -41,4 +41,82 @@ cd "$DOCKER_DIR" || {
+        echo "🔄 Starting missing dependent services:$missing_services"
+        docker compose -f docker-compose.yml up -d $missing_services
+    fi
+}
+
+# Wait for critical services to be healthy
+echo "Checking critical services..."
+check_service_health "api" 12  # 2 minutes with 10s intervals
+api_healthy=$?
+
+check_service_health "bisq2-api" 18  # 3 minutes with 10s intervals
+bisq2_healthy=$?
```

Shellcheck (0.11.0)

[warning] bisq2_healthy appears unused. Verify use (or export if used externally).

(SC2034)